src/actions/index.js

```diff
@@ -0,0 +1,6 @@
+//Sample actions
+export const FETCH_ARTICLES_SUCCESS = "FETCH_ARTICLES_SUCCESS";
+
```

Comment:
why is there a blank line here?

Reply:
@mubstimor That is to separate the variable declaration from the method.